C/c4Document.cc

```diff
@@ -39,17 +39,15 @@ using namespace litecore;
 
 
 C4Document::C4Document(C4Collection *collection, alloc_slice docID_)
-:_collection(asInternal(collection))
-,_docID(move(docID_))
-{
+        : _collection(asInternal(collection)), _docID(move(docID_)) {
     // Quick sanity test of the docID, but no need to scan for valid UTF-8 since we're not inserting.
     if (_docID.size < 1 || _docID.size > kMaxDocIDLength)
         error::_throw(error::BadDocID, "Invalid docID \"%.*s\"", SPLAT(_docID));
 
 #if DEBUG
     // Make sure that C4Document and C4Document_C line up so that the same object can serve as both:
-    auto asStruct = (C4Document_C*)this;
-    if (&_flags != &(asStruct)->flags || (void*)&_docID != (void*)&(asStruct)->docID) {
+    auto asStruct = (C4Document_C *)this;
+    if (&_flags != &(asStruct)->flags || (void *)&_docID != (void *)&(asStruct)->docID) {
```

Comment:
Putting spaces in a typecast like this looks awkward IMO, and makes it less clear that it's a single operation.

Reply:
Fixed with move to Clang-Format